auto_nag/scripts/b2g_query_creator.py

```diff
@@ -104,11 +105,15 @@ def createQueriesList(print_all, queries_dir):
 ]
 
 
-def cleanUp():
-    for file in os.listdir(queries_dir):
-        if file.startswith(str(datetime.date.today())):
-            os.remove(os.path.join(queries_dir, file))
-
+def cleanUp(queries_dir):
+    try:
+        for file in os.listdir(queries_dir):
+            if file.startswith(str(datetime.date.today())):
```

Comment:
by the way, it is not test related. 

It contains all the queries which we make; something like this in that folder 

queries : 2015-11-09_koi_regressions_unfixed,  2015-11-09_needinfo_sec_blockers , 2015-11-09_unfixed_koi_sec, 2015-11-09_needinfo_blockers, 2015-11-09_unfixed_koi

queries/2015-11-09_koi_regressions_unfixed contains:
query_name = 'Koi Blocker Bugs, Regressions'
query_url = 'https://bugzilla.mozilla.org//buglist.cgi?o5=nowordssubstr&f1=OP&f0=OP&f8=owner_idle_time&o2=equals&f4=OP&v5=fixed%20verified%20unaffected%20wontfix&j1=OR&f3=CP&f2=cf_blocking_b2g&bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&j4=OR&f5=cf_status_b2g_1_2&v8=-3d&f6=CP&v2=koi%2B&f7=CP&o8=greaterthan'
show_summary = '1'


since it is written in auto_nag/scripts/b2g_query_creator.py; we should write a test-methods for that also!


But we will clear all the files, instead of checking the names.